## rename-genbox-obj.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: rename genbox in obj

@DPATCH@
--- a/obj/cabin/bathroom
+++ b/obj/cabin/bathroom
@@ -1385,7 +1385,7 @@ bathfauc_mat ring bath.drain
 # Window
 #
 # xform -t 13.5 -4.75 37.75
-# genbox wframe_mat wframe.lower 33.5 5.75 2.25
+# genrbox wframe_mat wframe.lower 33.5 5.75 2.25
 
 wframe_mat polygon bath.wframe.lower.1540
 0
@@ -1441,7 +1441,7 @@ wframe_mat polygon bath.wframe.lower.645
 	-0.38083	14.12284	3.3012
 	0.09228	14.12284	3.3012
 # xform -t 44.5 -4.75 37.75
-# genbox wframe_mat wframe.north 2.25 5.75 44.5
+# genrbox wframe_mat wframe.north 2.25 5.75 44.5
 
 wframe_mat polygon bath.wframe.north.1540
 0
@@ -1497,7 +1497,7 @@ wframe_mat polygon bath.wframe.north.645
 	-0.38083	14.14341	6.77753
 	0.09228	14.14341	6.77753
 # xform -t 13.5 -4.75 80
-# genbox wframe_mat wframe.upper 33.5 5.75 2.25
+# genrbox wframe_mat wframe.upper 33.5 5.75 2.25
 
 wframe_mat polygon bath.wframe.upper.1540
 0
@@ -1553,7 +1553,7 @@ wframe_mat polygon bath.wframe.upper.645
 	-0.38083	14.12284	6.77753
 	0.09228	14.12284	6.77753
 # xform -t 13.5 -4.75 37.75
-# genbox wframe_mat wframe.south 2.25 5.75 44.5
+# genrbox wframe_mat wframe.south 2.25 5.75 44.5
 
 wframe_mat polygon bath.wframe.south.1540
 0
@@ -1869,7 +1869,7 @@ window_mat polygon bath.window.22
 
 # xform -e -n bath -rz -160 -s .08333 -t 8.34 17.2 0
 # xform -t .5 0 1
-# genbox blue_enamel door 27 1.375 78
+# genrbox blue_enamel door 27 1.375 78
 
 blue_enamel polygon bath.door.1540
 0
@@ -1972,7 +1972,7 @@ brass ring bath.outer_knobplate
 # Door Frame
 #
 # xform -t 6.75 99.5 80.25
-# genbox dframe_mat dframe.upper 32.5 6.75 2.25 -r .25
+# genrbox dframe_mat dframe.upper 32.5 6.75 2.25 -r .25
 
 dframe_mat polygon bath.dframe.upper.dhgc
 0
@@ -2164,7 +2164,7 @@ dframe_mat sphere bath.dframe.upper.o
 0
 4	8.73168	14.78108	6.77753	0.02057
 # xform -t 6.75 99.5 0
-# genbox dframe_mat dframe.south 2.25 6.75 82.5 -r .25
+# genrbox dframe_mat dframe.south 2.25 6.75 82.5 -r .25
 
 dframe_mat polygon bath.dframe.south.dhgc
 0
@@ -2356,7 +2356,7 @@ dframe_mat sphere bath.dframe.south.o
 0
 4	8.73168	17.27005	6.77753	0.02057
 # xform -t 37 99.5 0
-# genbox dframe_mat dframe.north 2.25 6.75 82.5 -r .25
+# genrbox dframe_mat dframe.north 2.25 6.75 82.5 -r .25
 
 dframe_mat polygon bath.dframe.north.dhgc
 0
@@ -4341,7 +4341,7 @@ marble polygon bath.vanity.sink.92
 	4.46921033945	17.035552	2.12384564754
 # edit the following:
 # xform -t 0 0 30
-# genbox marble sink 31 22 .75
+# genrbox marble sink 31 22 .75
 
 marble polygon bath.vanity.sink.1540
 0
@@ -4394,7 +4394,7 @@ marble polygon bath.vanity.sink.5137
 	5.572128	17.981772	2.4784
 	5.572128	17.981772	2.54011
 # xform -t 0 21.25 30.76
-# genbox marble sink_back 31 .75 3
+# genrbox marble sink_back 31 .75 3
 
 marble polygon bath.vanity.sink_back.1540
 0
@@ -4650,7 +4650,7 @@ zwood polygon bath.vanity.cab_right.8
 	5.489848	16.583012	0.01
 	5.530988	16.583012	0.01
 # xform -t .5 4.5 0
-# genbox xwood cab_base 30 .5 4
+# genrbox xwood cab_base 30 .5 4
 
 ywood polygon bath.vanity.cab_base.1540
 0
@@ -4706,7 +4706,7 @@ ywood polygon bath.vanity.cab_base.6457
 	5.530988	16.541872	0.33912
 	5.530988	16.583012	0.33912
 # xform -t .5 3.5 4
-# genbox zwood cab_front 30 .5 20.5 -r .1
+# genrbox zwood cab_front 30 .5 20.5 -r .1
 
 zwood polygon bath.vanity.cab_front.dhgc
 0
@@ -4898,7 +4898,7 @@ zwood sphere bath.vanity.cab_front.o
 0
 4	5.52276	16.492504	2.017632	0.008228
 # xform -t 0 -.5 0 -rx 15.25 -t .5 3 24.5
-# genbox xwood cab_front_top 30 .5 5.75 -r .1
+# genrbox xwood cab_front_top 30 .5 5.75 -r .1
 
 ywood polygon bath.vanity.cab_front_top.dhgc
 0
@@ -5090,7 +5090,7 @@ ywood sphere bath.vanity.cab_front_top.o
 0
 4	5.52276	16.2882352529	2.47220803984	0.008228
 # xform -t 2 2.95 5
-# genbox zwood cab_door 13.5 .5 18.5 -r .1
+# genrbox zwood cab_door 13.5 .5 18.5 -r .1
 
 zwood polygon bath.vanity.cab_door.dhgc
 0
@@ -5282,7 +5282,7 @@ zwood sphere bath.vanity.cab_door.o
 0
 4	4.28856	16.44725	1.935352	0.008228
 # xform -t 18 2.95 5
-# genbox xwood cab_drawer_lower 11 .5 5.75 -r .1
+# genrbox xwood cab_drawer_lower 11 .5 5.75 -r .1
 
 ywood polygon bath.vanity.cab_drawer_lower.dhgc
 0
@@ -5474,7 +5474,7 @@ ywood sphere bath.vanity.cab_drawer_lowe
 0
 4	5.39934	16.44725	0.886282	0.008228
 # xform -t 18 2.95 11.25
-# genbox xwood cab_drawer_middle 11 .5 5.75 -r .1
+# genrbox xwood cab_drawer_middle 11 .5 5.75 -r .1
 
 ywood polygon bath.vanity.cab_drawer_middle.dhgc
 0
@@ -5666,7 +5666,7 @@ ywood sphere bath.vanity.cab_drawer_midd
 0
 4	5.39934	16.44725	1.400532	0.008228
 # xform -t 18 2.95 17.75
-# genbox xwood cab_drawer_top 11 .5 5.75 -r .1
+# genrbox xwood cab_drawer_top 11 .5 5.75 -r .1
 
 ywood polygon bath.vanity.cab_drawer_top.dhgc
 0
@@ -5861,7 +5861,7 @@ ywood sphere bath.vanity.cab_drawer_top.
 #
 #	A mirrored wooden medicine cabinet.
 #
-# genbox zwood cabinet 4 46.5 27.5
+# genrbox zwood cabinet 4 46.5 27.5
 
 zwood polygon bath.mirror.cabinet.1540
 0
@@ -6301,7 +6301,7 @@ ywood cylinder bath.mirror.upper_right_f
 	4.78224	17.595056	5.944445
 	0.010285
 # xform -t 4 15.625 -.875
-# genbox ywood lower_middle_frame .875 15.25 1.75
+# genrbox ywood lower_middle_frame .875 15.25 1.75
 
 ywood polygon bath.mirror.lower_middle_frame.1540
 0
@@ -6365,7 +6365,7 @@ ywood cylinder bath.mirror.lower_middle_
 	4.771955	17.595056	3.825735
 	0.010285
 # xform -t 4 15.625 26.625
-# genbox ywood upper_middle_frame .875 15.25 1.75
+# genrbox ywood upper_middle_frame .875 15.25 1.75
 
 ywood polygon bath.mirror.upper_middle_frame.1540
 0
@@ -6437,7 +6437,7 @@ ywood cylinder bath.mirror.upper_middle_
 # A humble throne
 #
 # xform -t 7 5.5 0
-# genbox toilet_mat base0 15 9 1 -r .2
+# genrbox toilet_mat base0 15 9 1 -r .2
 
 toilet_mat polygon bath.toilet.base0.dhgc
 0
@@ -6718,7 +6718,7 @@ toilet_mat polygon bath.toilet.base1.7
 	1.30591	17.054065	0.05114
 	1.71731	17.054065	0.05114
 # xform -t 0 5.75 13
-# genbox toilet_mat base2 11 8.5 2 -r .2
+# genrbox toilet_mat base2 11 8.5 2 -r .2
 
 toilet_mat polygon bath.toilet.base2.dhgc
 0
--- a/obj/cabin/bdfurn.rad
+++ b/obj/cabin/bdfurn.rad
@@ -10,9 +10,9 @@
 # The Cabin's Bed.
 #
 
-!genbox rose_spread bed 6 4.5 1.733 -r .2 | xform -t .08333 1.5 .1
+!genrbox rose_spread bed 6 4.5 1.733 -r .2 | xform -t .08333 1.5 .1
 
-!genbox rose_spread pillows 1.333 4.5 .5 -r .2 | xform -t .08333 1.5 1.833
+!genrbox rose_spread pillows 1.333 4.5 .5 -r .2 | xform -t .08333 1.5 1.833
 
 
 
--- a/obj/cabin/cabin
+++ b/obj/cabin/cabin
@@ -8,12 +8,12 @@
 
 # Cabin Foundation
 
-!genbox concrete foundation 27.8 18.8 3 | xform -t -.4 -.4 -3.0001 
+!genrbox concrete foundation 27.8 18.8 3 | xform -t -.4 -.4 -3.0001 
 
 
 # Cabin Porch
 
-!genbox xybrick porch 6 3 3 | xform -t 18 18.5 -3.3
+!genrbox xybrick porch 6 3 3 | xform -t 18 18.5 -3.3
 
 
 # Cabin Floor
@@ -54,7 +54,7 @@ wood_floor polygon floor
 # The Chimney
 
 # xform -t 26.25 5.5 -3
-# genbox fireplace_stone chimney 2 3 22
+# genrbox fireplace_stone chimney 2 3 22
 
 xzbrick polygon south.chimney
 0
@@ -121,7 +121,7 @@ yxbrick polygon top.chimney
 
 
 # xform -t 26.25 6 1.25
-# genbox cinderblock fireplace 1.5 2 1.5 -i
+# genrbox cinderblock fireplace 1.5 2 1.5 -i
 
 cinderblock polygon fireplace.1540
 0
@@ -321,7 +321,7 @@ cinderblock polygon chimney_stack.13
 # Hearth
 
 # xform -t 25 4 0
-# genbox fireplace_stone hearth 2 6 1.25
+# genrbox fireplace_stone hearth 2 6 1.25
 
 xzbrick polygon hearth.south
 0
@@ -417,9 +417,9 @@ wood_panel polygon nbed_wall
 # Bedroom Closet
 
 # xform -t 6 11 0
-# genbox wood_panel closet 5 2.75 6.5 -i
+# genrbox wood_panel closet 5 2.75 6.5 -i
 
-# The opening to the closet created by genbox is commented out here,
+# The opening to the closet created by genrbox is commented out here,
 # to provide a closet door opening to the closet.
 #wood_panel polygon closet.1540
 #0
@@ -489,7 +489,7 @@ light_wood cylinder closet_dowel
 
 # Closet Shelf
 
-!genbox light_wood closet_shelf 5 1.5 .08333 | xform -t 6 12.25 5.33
+!genrbox light_wood closet_shelf 5 1.5 .08333 | xform -t 6 12.25 5.33
 
 # Interior Livingroom Wall
 
--- a/obj/cabin/chair.norm
+++ b/obj/cabin/chair.norm
@@ -5,12 +5,12 @@
 # (to the sitter's left, and back).
 #
 
-!genbox brown_tweed back 30 5 33 -r 1
+!genrbox brown_tweed back 30 5 33 -r 1
 
-!genbox brown_tweed left_arm 5 17 25 -r 1 | xform -t 0 5 0
+!genrbox brown_tweed left_arm 5 17 25 -r 1 | xform -t 0 5 0
 
-!genbox brown_tweed right_arm 5 17 25 -r 1 | xform -t 25 5 0
+!genrbox brown_tweed right_arm 5 17 25 -r 1 | xform -t 25 5 0
 
-!genbox brown_tweed base 20 17 12 -r 1 | xform -t 5 5 0
+!genrbox brown_tweed base 20 17 12 -r 1 | xform -t 5 5 0
 
-!genbox brown_tweed cushion 20 17 5 -r 1 | xform -t 5 5 12
+!genrbox brown_tweed cushion 20 17 5 -r 1 | xform -t 5 5 12
--- a/obj/cabin/door.norm
+++ b/obj/cabin/door.norm
@@ -6,7 +6,7 @@
 # The materials door_paint & brass need to be defined.
 #
 
-!genbox door_paint door 35 1.375 76 | xform -t .5 0 1
+!genrbox door_paint door 35 1.375 76 | xform -t .5 0 1
 
 brass sphere inner_doorknob
 0
--- a/obj/cabin/dresser.norm
+++ b/obj/cabin/dresser.norm
@@ -108,7 +108,7 @@ xpine polygon cabinet.top
                  30                 20                 34
 
 
-!genbox zpine mirror_back 30 1 36 -r .25 | xform -t 0 0 34
+!genrbox zpine mirror_back 30 1 36 -r .25 | xform -t 0 0 34
 
 # mirror polygon dresser_mirror 
 # 0
@@ -126,7 +126,7 @@ xpine polygon cabinet.top
 # The three dresser drawers.
 # xform -n drawer.0 -t 1 20 2.5 -i 0 -t 0 0 10.5
 
-!genbox xpine drawer_front 28 .75 10 -r .25  | xform -e -n drawer.0 -t 1 20 2.5 -i 0 -t 0 0 10.5
+!genrbox xpine drawer_front 28 .75 10 -r .25  | xform -e -n drawer.0 -t 1 20 2.5 -i 0 -t 0 0 10.5
 # xform -n handle.0 -t 4 .75 5 -i 0 -t 15 0 0
 
 brass cylinder drawer.0.handle.0.right_stem
@@ -189,7 +189,7 @@ brass cylinder drawer.0.handle.1.left_st
                0.25
 # xform -n drawer.1 -t 1 20 2.5 -i 1 -t 0 0 10.5
 
-!genbox xpine drawer_front 28 .75 10 -r .25  | xform -e -n drawer.1 -t 1 20 2.5 -i 1 -t 0 0 10.5
+!genrbox xpine drawer_front 28 .75 10 -r .25  | xform -e -n drawer.1 -t 1 20 2.5 -i 1 -t 0 0 10.5
 # xform -n handle.0 -t 4 .75 5 -i 0 -t 15 0 0
 
 brass cylinder drawer.1.handle.0.right_stem
@@ -252,7 +252,7 @@ brass cylinder drawer.1.handle.1.left_st
                0.25
 # xform -n drawer.2 -t 1 20 2.5 -i 2 -t 0 0 10.5
 
-!genbox xpine drawer_front 28 .75 10 -r .25  | xform -e -n drawer.2 -t 1 20 2.5 -i 2 -t 0 0 10.5
+!genrbox xpine drawer_front 28 .75 10 -r .25  | xform -e -n drawer.2 -t 1 20 2.5 -i 2 -t 0 0 10.5
 # xform -n handle.0 -t 4 .75 5 -i 0 -t 15 0 0
 
 brass cylinder drawer.2.handle.0.right_stem
--- a/obj/cabin/furniture
+++ b/obj/cabin/furniture
@@ -16,24 +16,24 @@
 # The Cabin's Bed.
 #
 
-!genbox rose_spread bed 6 4.5 1.733 -r .2 | xform -t .08333 1.5 .1
+!genrbox rose_spread bed 6 4.5 1.733 -r .2 | xform -t .08333 1.5 .1
 
-!genbox rose_spread pillows 1.333 4.5 .5 -r .2 | xform -t .08333 1.5 1.833
+!genrbox rose_spread pillows 1.333 4.5 .5 -r .2 | xform -t .08333 1.5 1.833
 
 
 #
 # The Table.
 #
 
-!genbox ypine tabletop 2 2 .1667 | xform -t 15 1 2 
+!genrbox ypine tabletop 2 2 .1667 | xform -t 15 1 2 
 
-!genbox zpine leg1 .1667 .1667 2 | xform -t 15 1 0
+!genrbox zpine leg1 .1667 .1667 2 | xform -t 15 1 0
 
-!genbox zpine leg2 .1667 .1667 2 | xform -t 15 2.8333 0
+!genrbox zpine leg2 .1667 .1667 2 | xform -t 15 2.8333 0
 
-!genbox zpine leg3 .1667 .1667 2 | xform -t 16.8333 2.8333 0
+!genrbox zpine leg3 .1667 .1667 2 | xform -t 16.8333 2.8333 0
 
-!genbox zpine leg4 .1667 .1667 2 | xform -t 16.8333 1 0   
+!genrbox zpine leg4 .1667 .1667 2 | xform -t 16.8333 1 0   
 
 
 # The Dresser in Bedroom
--- a/obj/cabin/lvfurn.rad
+++ b/obj/cabin/lvfurn.rad
@@ -14,15 +14,15 @@
 # The Table.
 #
 
-!genbox ypine tabletop 2 2 .1667 | xform -t 15 1 2 
+!genrbox ypine tabletop 2 2 .1667 | xform -t 15 1 2 
 
-!genbox zpine leg1 .1667 .1667 2 | xform -t 15 1 0
+!genrbox zpine leg1 .1667 .1667 2 | xform -t 15 1 0
 
-!genbox zpine leg2 .1667 .1667 2 | xform -t 15 2.8333 0
+!genrbox zpine leg2 .1667 .1667 2 | xform -t 15 2.8333 0
 
-!genbox zpine leg3 .1667 .1667 2 | xform -t 16.8333 2.8333 0
+!genrbox zpine leg3 .1667 .1667 2 | xform -t 16.8333 2.8333 0
 
-!genbox zpine leg4 .1667 .1667 2 | xform -t 16.8333 1 0   
+!genrbox zpine leg4 .1667 .1667 2 | xform -t 16.8333 1 0   
 
 
 # The Coatrack in Livingroom
--- a/obj/cabin/sofa.norm
+++ b/obj/cabin/sofa.norm
@@ -4,17 +4,17 @@
 # The scale is in inches.
 #
 
-!genbox brown_tweed back 60 5 33 -r 1 | xform -t 5 0 0
+!genrbox brown_tweed back 60 5 33 -r 1 | xform -t 5 0 0
 
-!genbox brown_tweed left_arm 5 22 25 -r 1
+!genrbox brown_tweed left_arm 5 22 25 -r 1
 
-!genbox brown_tweed right_arm 5 22 25 -r 1 | xform -t 65 0 0
+!genrbox brown_tweed right_arm 5 22 25 -r 1 | xform -t 65 0 0
 
-!genbox brown_tweed base 60 17 12 -r 1 | xform -t 5 5 0
+!genrbox brown_tweed base 60 17 12 -r 1 | xform -t 5 5 0
 
-!genbox brown_tweed left_cushion 20 17 5 -r 1 | xform -t 5 5 12
+!genrbox brown_tweed left_cushion 20 17 5 -r 1 | xform -t 5 5 12
 
-!genbox brown_tweed middle_cushion 20 17 5 -r 1 | xform -t 25 5 12
+!genrbox brown_tweed middle_cushion 20 17 5 -r 1 | xform -t 25 5 12
 
-!genbox brown_tweed right_cushion 20 17 5 -r 1 | xform -t 45 5 12
+!genrbox brown_tweed right_cushion 20 17 5 -r 1 | xform -t 45 5 12
 
--- a/obj/misc/examp
+++ b/obj/misc/examp
@@ -7,7 +7,7 @@ void plastic sky_blue
 0
 5	.6	.6	.7	0	0
 
-!genbox sky_blue room 10 5 8 -i
+!genrbox sky_blue room 10 5 8 -i
 
 void metal2 gold
 4 1 1 1 .
--- a/obj/office/book.norm
+++ b/obj/office/book.norm
@@ -3,11 +3,11 @@
 #  The cover material must be defined as cover_mat
 #
 
-!genbox cover_mat back_cover 6.5 .125 9.5
+!genrbox cover_mat back_cover 6.5 .125 9.5
 
-!genbox cover_mat front_cover 6.5 .125 9.5 | xform -t 0 1.25 0
+!genrbox cover_mat front_cover 6.5 .125 9.5 | xform -t 0 1.25 0
 
-!genbox cover_mat bind_cover .125 1.125 9.5 | xform -t -.125 .125 0
+!genrbox cover_mat bind_cover .125 1.125 9.5 | xform -t -.125 .125 0
 
 void plastic page_mat
 0
--- a/obj/office/chair2.norm
+++ b/obj/office/chair2.norm
@@ -2,10 +2,10 @@
 #  A straight chair
 #
 
-!genbox leg_material leg.right.rear .75 26 1.5
-!genbox leg_material leg.left.rear .75 26 1.5 | xform -t 24 0 0
-!genbox leg_material leg.right.front .75 26 1.5 | xform -t 0 0 20
-!genbox leg_material leg.left.front .75 26 1.5 | xform -t 24 0 20
+!genrbox leg_material leg.right.rear .75 26 1.5
+!genrbox leg_material leg.left.rear .75 26 1.5 | xform -t 24 0 0
+!genrbox leg_material leg.right.front .75 26 1.5 | xform -t 0 0 20
+!genrbox leg_material leg.left.front .75 26 1.5 | xform -t 24 0 20
 
 leg_material cylinder strut
 0
@@ -63,6 +63,6 @@ arm_material ring arm.cap.left.front
 	0	0	1
 	0	.75
 
-!genbox cushion_material seat 23.25 3 22 -r 1 | xform -t .75 18 0
+!genrbox cushion_material seat 23.25 3 22 -r 1 | xform -t .75 18 0
 
-!genbox cushion_material back 23.25 3 10 -r 1 | xform -rx 80 -t .75 33 0
+!genrbox cushion_material back 23.25 3 10 -r 1 | xform -rx 80 -t .75 33 0
--- a/obj/office/cornerdesk.norm
+++ b/obj/office/cornerdesk.norm
@@ -297,8 +297,8 @@ ypine cylinder paneledge_rightrear
                 7.5                2.2                  0
                0.05
 
-!genbox grey_plastic top_drawer 1.4 .5 1.1 -r .05 | xform -t .1 1.6 4.375
+!genrbox grey_plastic top_drawer 1.4 .5 1.1 -r .05 | xform -t .1 1.6 4.375
 
-!genbox grey_plastic middle_drawer 1.4 .5 1.1 -r .05 | xform -t .1 1.05 4.375
+!genrbox grey_plastic middle_drawer 1.4 .5 1.1 -r .05 | xform -t .1 1.05 4.375
 
-!genbox grey_plastic bottom_drawer 1.4 1 1.1 -r .05 | xform -t .1 0 4.375
+!genrbox grey_plastic bottom_drawer 1.4 1 1.1 -r .05 | xform -t .1 0 4.375
--- a/obj/office/openbook.norm
+++ b/obj/office/openbook.norm
@@ -35,11 +35,11 @@ page2_text plastic page2_mat
 0
 5 .7 .7 .7 .05 .08
 
-!genbox cover_mat back_cover 6.5 .125 9.5 | xform -rz -1.1 -t 0 .125 0
+!genrbox cover_mat back_cover 6.5 .125 9.5 | xform -rz -1.1 -t 0 .125 0
 
-!genbox cover_mat front_cover 6.5 .125 9.5 | xform -t -6.5 0 0 -rz 1.1 -t -1.125 .125 0
+!genrbox cover_mat front_cover 6.5 .125 9.5 | xform -t -6.5 0 0 -rz 1.1 -t -1.125 .125 0
 
-!genbox cover_mat bind_cover 1.125 .125 9.5 | xform -t -1.125 0 0
+!genrbox cover_mat bind_cover 1.125 .125 9.5 | xform -t -1.125 0 0
 
 !genprism page1_mat page1 pageverts -l 0 0 9.3 | xform -t 0 0 .1
 
--- a/obj/texture/room2
+++ b/obj/texture/room2
@@ -17,7 +17,7 @@ void metal brass
 0
 5	0.68	0.27	0.002	0.95	0
 
-!genbox brass track 70 2 3 | xform -t 35 94 88.5
+!genrbox brass track 70 2 3 | xform -t 35 94 88.5
 
 !xform -n flood1 -s 12 -rx -90 -rx 20 -t 40 84 90 flood
 !xform -n flood2 -s 12 -rx -90 -rx 25 -t 60 84 90 flood
--- a/obj/virtual/draft_ang.rad
+++ b/obj/virtual/draft_ang.rad
@@ -64,7 +64,7 @@ void plastic pale_green
 # Original units in inches
 #
 # xform -t 0 0 .5
-# genbox x-aluminum leg 26 1 2
+# genrbox x-aluminum leg 26 1 2
 
 x-aluminum polygon leg.0.leg.1540
 0
@@ -152,7 +152,7 @@ chrome cone leg.0.frontfoot
              1.5494       0.6582312718             0.0127
              0.0127            0.00635
 # xform -t 11.75 0 2.5
-# genbox z-aluminum stand_support 2.5 .5 44.25
+# genrbox z-aluminum stand_support 2.5 .5 44.25
 
 z-aluminum polygon leg.0.stand_support.1540
 0
@@ -407,7 +407,7 @@ darker cylinder leg.0.knob_stem
 # Original units in inches
 #
 # xform -t 0 0 .5
-# genbox x-aluminum leg 26 1 2
+# genrbox x-aluminum leg 26 1 2
 
 x-aluminum polygon leg.1.leg.1540
 0
@@ -495,7 +495,7 @@ chrome cone leg.1.frontfoot
              0.0508       0.6582312718             0.0127
              0.0127            0.00635
 # xform -t 11.75 0 2.5
-# genbox z-aluminum stand_support 2.5 .5 44.25
+# genrbox z-aluminum stand_support 2.5 .5 44.25
 
 z-aluminum polygon leg.1.stand_support.1540
 0
@@ -744,7 +744,7 @@ darker cylinder leg.1.knob_stem
              0.0381       0.3407312718             1.1176
              0.0127
 # xform -t 11.75 .5 8
-# genbox y-aluminum kick_stand .5 59 3
+# genrbox y-aluminum kick_stand .5 59 3
 
 y-aluminum polygon kick_stand.1540
 0
@@ -814,7 +814,7 @@ darker cylinder axel
 # Original units in inches
 #
 # xform -t -18 0 0
-# genbox pale_green top 36 60 .75
+# genrbox pale_green top 36 60 .75
 
 pale_green polygon table.side1
 0
@@ -1378,7 +1378,7 @@ darker polygon table.adjust2.22
             0.03556     0.420324943067      1.16555593701
              0.0381     0.420324943067      1.16555593701
 # xform -t -17 .5 -1
-# genbox darker left_support 34 1 1
+# genrbox darker left_support 34 1 1
 
 darker polygon table.left_support.1540
 0
@@ -1434,7 +1434,7 @@ darker polygon table.left_support.6457
              1.5494      0.70054156705     0.946626716382
               1.524      0.70054156705     0.946626716382
 # xform -t -17 58.5 -1
-# genbox darker right_support 34 1 1
+# genrbox darker right_support 34 1 1
 
 darker polygon table.right_support.1540
 0
--- a/obj/virtual/draft_flat.rad
+++ b/obj/virtual/draft_flat.rad
@@ -64,7 +64,7 @@ void plastic pale_green
 # Original units in inches
 #
 # xform -t 0 0 .5
-# genbox x-aluminum leg 26 1 2
+# genrbox x-aluminum leg 26 1 2
 
 x-aluminum polygon leg.0.leg.1540
 0
@@ -152,7 +152,7 @@ chrome cone leg.0.frontfoot
              1.5494       0.6582312718             0.0127
              0.0127            0.00635
 # xform -t 11.75 0 2.5
-# genbox z-aluminum stand_support 2.5 .5 44.25
+# genrbox z-aluminum stand_support 2.5 .5 44.25
 
 z-aluminum polygon leg.0.stand_support.1540
 0
@@ -407,7 +407,7 @@ darker cylinder leg.0.knob_stem
 # Original units in inches
 #
 # xform -t 0 0 .5
-# genbox x-aluminum leg 26 1 2
+# genrbox x-aluminum leg 26 1 2
 
 x-aluminum polygon leg.1.leg.1540
 0
@@ -495,7 +495,7 @@ chrome cone leg.1.frontfoot
              0.0508       0.6582312718             0.0127
              0.0127            0.00635
 # xform -t 11.75 0 2.5
-# genbox z-aluminum stand_support 2.5 .5 44.25
+# genrbox z-aluminum stand_support 2.5 .5 44.25
 
 z-aluminum polygon leg.1.stand_support.1540
 0
@@ -744,7 +744,7 @@ darker cylinder leg.1.knob_stem
              0.0381       0.3407312718             1.1176
              0.0127
 # xform -t 11.75 .5 8
-# genbox y-aluminum kick_stand .5 59 3
+# genrbox y-aluminum kick_stand .5 59 3
 
 y-aluminum polygon kick_stand.1540
 0
@@ -814,7 +814,7 @@ darker cylinder axel
 # Original units in inches
 #
 # xform -t -18 0 0
-# genbox pale_green top 36 60 .75
+# genrbox pale_green top 36 60 .75
 
 pale_green polygon table.side1
 0
@@ -1378,7 +1378,7 @@ darker polygon table.adjust2.22
             0.03556       0.4169312718             1.2192
              0.0381       0.4169312718             1.2192
 # xform -t -17 .5 -1
-# genbox darker left_support 34 1 1
+# genrbox darker left_support 34 1 1
 
 darker polygon table.left_support.1540
 0
@@ -1434,7 +1434,7 @@ darker polygon table.left_support.6457
              1.5494       0.7725312718             1.2192
               1.524       0.7725312718             1.2192
 # xform -t -17 58.5 -1
-# genbox darker right_support 34 1 1
+# genrbox darker right_support 34 1 1
 
 darker polygon table.right_support.1540
 0
--- a/obj/virtual/environ.rad
+++ b/obj/virtual/environ.rad
@@ -61,7 +61,7 @@ void plastic neigh1.roof_mat
 5 .08 .08 .08 0 0
 
 # xform -t 4 -20 0
-# genbox neigh1.wall_mat neigh1 15 15 10
+# genrbox neigh1.wall_mat neigh1 15 15 10
 
 neigh1.xzwall_mat polygon neigh1.1540
 0
--- a/obj/virtual/illumoff.rad
+++ b/obj/virtual/illumoff.rad
@@ -52,7 +52,7 @@ void glass window_mat
 # The building surfaces
 #
 # xform -t -.25 -.25 0
-# genbox build_mat building 6.5 8.5 3.45
+# genrbox build_mat building 6.5 8.5 3.45
 
 build_mat polygon building.1540
 0
@@ -107,7 +107,7 @@ build_mat polygon building.6457
 #
 # The room surfaces
 #
-# genbox wall_mat wall 6 8 3.2 -i
+# genrbox wall_mat wall 6 8 3.2 -i
 
 wall_mat polygon wall.south
 0
@@ -201,7 +201,7 @@ wall_mat polygon wall.east
 # The light shelf
 #
 # xform -t .25 -1.5 2.175
-# genbox shelfbot_mat shelf 5.5 2.5 .05
+# genrbox shelfbot_mat shelf 5.5 2.5 .05
 
 shelfbot_mat polygon shelf.1540
 0
--- a/obj/virtual/plainoff.rad
+++ b/obj/virtual/plainoff.rad
@@ -41,7 +41,7 @@ void glass window_mat
 # The building surfaces
 #
 # xform -t -.25 -.25 0
-# genbox build_mat building 6.5 8.5 3.45
+# genrbox build_mat building 6.5 8.5 3.45
 
 build_mat polygon building.1540
 0
@@ -97,7 +97,7 @@ build_mat polygon building.6457
 #
 # The room surfaces
 #
-# genbox wall_mat wall 6 8 3.2 -i
+# genrbox wall_mat wall 6 8 3.2 -i
 
 wall_mat polygon wall.south
 0
@@ -157,7 +157,7 @@ wall_mat polygon wall.east
 # The light shelf
 #
 # xform -t .25 -1.5 2.175
-# genbox shelfbot_mat shelf 5.5 2.5 .05
+# genrbox shelfbot_mat shelf 5.5 2.5 .05
 
 shelfbot_mat polygon shelf.1540
 0
--- a/obj/virtual/stool.rad
+++ b/obj/virtual/stool.rad
@@ -932,7 +932,7 @@ leg_mat cylinder support
              0.3175             0.3175             0.7112
             0.01905
 # xform -t -2.5 -2 28
-# genbox leg_mat spring_mechanism 6 4 2.5
+# genrbox leg_mat spring_mechanism 6 4 2.5
 
 leg_mat polygon spring_mechanism.1540
 0
@@ -988,7 +988,7 @@ leg_mat polygon spring_mechanism.6457
              0.2667             0.2286             0.7747
              0.3683             0.2286             0.7747
 # xform -t 3.5 -.875 29.5
-# genbox chrome_mat back_support.lower 10 1.75 .15
+# genrbox chrome_mat back_support.lower 10 1.75 .15
 
 chrome_mat polygon back_support.lower.1540
 0
@@ -1044,7 +1044,7 @@ chrome_mat polygon back_support.lower.64
            0.295275            -0.0254            0.75311
            0.339725            -0.0254            0.75311
 # xform -t 13.5 -.875 29.5
-# genbox chrome_mat back_support.upper .15 1.75 15
+# genrbox chrome_mat back_support.upper .15 1.75 15
 
 chrome_mat polygon back_support.upper.1540
 0
@@ -1100,7 +1100,7 @@ chrome_mat polygon back_support.upper.64
            0.295275           -0.02921             1.1303
            0.339725           -0.02921             1.1303
 # xform -t -10 -8 30.5
-# genbox seat_mat seat 18 16 3 -r 1
+# genrbox seat_mat seat 18 16 3 -r 1
 
 seat_mat polygon seat.dhgc
 0
@@ -1292,7 +1292,7 @@ seat_mat sphere seat.o
 0
 4             0.4953             0.1397             0.8255             0.0254
 # xform -t -1.5 -7 -5 -ry 15 -t 13.5 0 44.5
-# genbox seat_mat back 1.5 14 10 -r .5
+# genrbox seat_mat back 1.5 14 10 -r .5
 
 seat_mat polygon back.dhgc
 0
